match/lib/match/options.rb

```diff
@@ -292,6 +292,11 @@ def self.available_options
                                      description: "Renew the provisioning profiles if the certificate count on the developer portal has changed. Works only for the 'development' provisioning profile type. Requires 'include_all_certificates' option to be 'true'",
                                      type: Boolean,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :renew_expired_certs,
+                                     env_name: "MATCH_RENEW_EXPIRED_CERTS",
+                                     description: "Automatically renew expired certificates. Notice: to renew developer_id and developer_id_installer certificates you must login with Account Holder account",
+                                     type: Boolean,
+                                     default_value: false),
```

Comment:
I understand this was probably used to keep the previous behavior the same, but are there risks or any reason why someone would not want this feature enabled by default? I think it's also our responsibility (as _fastlane_ developers/contributors) to provide sensible defaults and this seems to be one of those cases :) this feature is too good to be left up for developers to discover it on their own and then have to add +1 LOC to enable it 😅

Reply:
Great suggestion. The default value is `true` now.